presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -144,6 +144,40 @@ If an error similar to the following example is displayed, this is probably beca
     	at org.projectnessie.client.http.impl.jdk8.UrlConnectionRequest.executeRequest(UrlConnectionRequest.java:71)
     	... 42 more
 
+REST catalog
+^^^^^^^^^^^^^^
```

Comment:
```suggestion
^^^^^^^^^^^^
```
Heading formatting should exactly match the characters in the preceding line, otherwise warnings are generated during build.